docker/Dockerfile.ubuntu.xenial

```diff
@@ -69,6 +67,23 @@ RUN apt-get update && apt-get -y install \
   softhsm2 \
   valgrind
 
+
+WORKDIR /cmake
+RUN wget https://github.com/Kitware/CMake/releases/download/v3.14.4/cmake-3.14.4.tar.gz -nv
```

Comment:
We might be stuck with having to compile boost but it feels like it should be easier to live with an older CMake version. Do you know what's blocking us from being compatible with CMake on Xenial?

Reply:
The CMake version lower than 3.8 does not properly support boost 1.64 or higher, it's related to Finding an installed library.

Reply:
Ah gotcha, I forgot about that, makes sense.